odata-csdl-validator/odata_validator.py

```diff
@@ -968,17 +968,31 @@ def _check_annotations(self):
             SchemaError: If an annotation is illegally applying a term to this element.
         """
 
+        # ODATA Team says that the AppliesTo Attribute is more of a guideline than a rule.
+        # From the CSDL Spec: As the intended usage may evolve over time, clients SHOULD be prepared for any annotation
+        # to be applied to any element.
+
+        # This definition inserts expected usage based on modeling patterns in Redfish
+        applies_to_mod_set = {
```

Comment:
Would like to discuss this a bit; we've avoided Redfish-specific definitions in this tool since it's intended to be a generic CSDL validator. Should we move something like this to a config file?

Reply:
2/24: Will add a config file option to pick up these definitions